packages/ui-kit/src/components/Card/Card.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
What is the criteria for marking something `'use client'` or not? Do we just need to do it for components that would use client-side events, like `onclick`, or for other things, too? Basically, I'm wondering what in Card requires us to mark it `'use client'`?

Reply:
Any component that uses state will require being a client component, in the case of card it uses the `themeSettings` which comes from a context